.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -32,12 +32,10 @@ _Describe what should have happened_.
 
 _Describe what actually happened instead_.
```

Comment:
```suggestion
_Describe what actually happened_.
```

Reply:
I have another change (to existing text, so I couldn't figure out how to use the "suggested change" feature):

_NOTE: If your issue is not reproducible using the dreamhouse-lwc repo (i.e. because it requires specific metadata or files), you must provide a link to a simple Salesforce project repository with a script to set up a scratch org that reproduces your problem._